src/fragments/guides/storage/transfer-accelaration.mdx

```diff
@@ -0,0 +1,43 @@
+When you need leverage S3's transfer accelaration on the content bucket, you can override the storage resource for this configuration and then leveraging the `useAccelerateEndpoint` parameter to use the accelarated S3 endpoint. This will enable fast, easy, and secure transfers of files over long distances between your client and the S3 bucket.
+
+## Override storage resource
+
+Start by overriding storage resource to enable Transfer Accelaration on the content bucket.
```

Comment:
Start by overriding your storage resources to enable Transfer Acceleration on your S3 bucket.

Reply:
Addressed the reference from _content_ bucket to _your S3_ bucket